app/models/team.rb

```diff
@@ -49,20 +49,6 @@ def self.all_official
     ]
   end
 
-  # Councils are recognized by Motion "20.2019.0":
-  #  https://documents.worldcubeassociation.org/documents/motions/20.2019.0%20-%20Councils.pdf
-  # Motions starting with "20.YYYY.N" define these councils:
-  #  https://www.worldcubeassociation.org/documents
```

Comment:
Do we really need to place the motion link in our code? The main reason why I don't like to have it is because we don't have a permalink for any motions. For example, the link mentioned in the comment no longer exist. If we add the latest link, if the motion is modified again in next cycle, we again have to change it, which is not so easy to track.

Reply:
I'm mostly talking about a reference to the Motions, where the notion of "Motion 20.2019.0 exists" is definitely permanent. Whether or not to include an actual URL is up for debate, but in my opinion it is "stable enough" to warrant inclusion. If you feel really bad about the URL, skip it. But at least include information about the Motion itself.

Reply:
Okay, I've added it in user_group.rb now, because I couldn't find a better place. Please let me know if there is a better place. Also please let me know if the text has to be changed.